### PR TITLE
UFORGE-5723 Deployment fails due to NIC settings

### PR DESCRIPTION
**Pull request by yunano**

### CODE REVIEW

File: admin/en/source/pages/config/modify-ip.rst [hunk 1]
```diff
@@ -13,7 +13,9 @@ There are three external URL endpoints for the UForge platform, namely:
 
 .. image:: /images/external-endpoints.png
 
-These URL endpoints are automatically created based on the external hostname provided during the initial configuration of the UForge platform (see :ref:`configure-uforge`).  These URL endpoints can be changed by updating certain variables in the ``/etc/UShareSoft/uforge/uforge.conf`` file.
+These URL endpoints can be changed by updating certain variables in the ``/etc/UShareSoft/uforge/uforge.conf`` file.
```
Comment (emuus) [inline admin/en/source/pages/config/modify-ip.rst]: Please leave this paragraph as is. The re-write does not add to the clarity.

File: admin/en/source/pages/config/modify-ip.rst [hunk 2]
```diff
@@ -23,7 +25,7 @@ The URL endpoint for direct REST web service access is constructed using the fol
 
 	https://<UFORGE_PROXY_INFOS>/<UFORGE_API_ROOT_CONTEXT>
 
-The download URL endpoint is constructed using the ``UFORGE_IAAS_DOWNLOAD_URL`` variable.
+The download URL endpoint is constructed using the ``UFORGE_IAAS_DOWNLOAD_URL`` variable, which is automatically created based on the external hostname provided during the initial configuration of the UForge platform.
```
Comment (emuus) [inline admin/en/source/pages/config/modify-ip.rst]: Reject this change. It simply repeats information from previous paragraph

File: admin/en/source/pages/install/configure-uforge.rst [hunk 3]
```diff
@@ -35,12 +35,14 @@ Fill in the wizard, note that all the fields are mandatory.
 
 	10. Enter the Web Server IP address, external hostname and the database IP address.
 
-		.. note:: The external hostname is used to construct two external URL endpoints.  The first as the external URL endpoint of the user interface, and the other for REST API calls and command-line usage.  The external hostname should normally be a fully qualified hostname.  For example, if the external hostname is uf.example.com, then the following URL endpoints are created:
+		.. note:: Regardless these entering, two external URL endpoints are determined from one of the IP addresses of UI server. The first as the external URL endpoint of the user interface, and the other for REST API calls and command-line usage. For example, if the address is 192.0.2.2, then the following URL endpoints are created:
```
Comment (emuus) [inline admin/en/source/pages/install/configure-uforge.rst]: Reject the rewrite of the paragraph.

File: admin/en/source/pages/install/configure-uforge.rst [hunk 4]
```diff
@@ -35,12 +35,14 @@ Fill in the wizard, note that all the fields are mandatory.
...
 
-			* User interface URL endpoint: https://uf.example.com/uforge
-			* Command-line URL endpoint: https://uf.example.com/apis
+			* User interface URL endpoint: https://192.0.2.2/uforge
+			* Command-line URL endpoint: https://192.0.2.2/apis
 
-			These external URL endpoints can be changed after the initial configuration is complete, refer to :ref:`modify-external-endpoints` for more information.
+			The external hostname is used to construct the download URL endpoint and should normally be a fully qualified hostname.
```
Comment (emuus) [inline admin/en/source/pages/install/configure-uforge.rst]: ok

File: admin/en/source/pages/config/modify-ip.rst (hunk 1 above)
Comment (yunano) [inline admin/en/source/pages/config/modify-ip.rst]: I will re-submit it after returning.

File: admin/en/source/pages/config/modify-ip.rst (hunk 2 above)
Comment (yunano) [inline admin/en/source/pages/config/modify-ip.rst]: same above

**Pull request merged**